server/config/config.env

```diff
@@ -0,0 +1,8 @@
+
+NODE_ENV=development  
+
+PORT=8080          
+
+MONGO_URI=mongodb+srv://tofek98:TDWA5GK4wFYi8kAY@test.imfukcq.mongodb.net/
```

Comment:
don't upload keys or passwords to github!
this should be in gitignore

Reply:
FIXED